### PR TITLE
Add Update main databox repo documentation 

### PR DESCRIPTION
**Pull request by Toshbrown**
Contributes to: https://github.com/me-box/databox/issues/102

### CODE REVIEW

**Comment by pooyadav** [comment]
I think this would go in documents.

**Review by pooyadav — CHANGES_REQUESTED** [review]
There are few typos - also I feel it should go in documents ( as we have all documents are there) and readme.md should be updated accordingly

**Comment by Toshbrown** [comment]
I'm happy for these to be moved (though it is the convention to have them here).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Contributing to Databox
+
+The databox project welcomes contributions via pull requests. This document 
+sets out the process for contibuting. 
+
+## Reporting issues
+
+Found a peroblem? Got a feature request? Then plese open an issue on the main
```
Comment (mor1) [inline CONTRIBUTING.md]: `peroblem`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Contributing to Databox
+
+The databox project welcomes contributions via pull requests. This document 
+sets out the process for contibuting. 
+
+## Reporting issues
+
+Found a peroblem? Got a feature request? Then plese open an issue on the main 
+[databox repository](https://github.com/me-box/databox/). First check that
```
Comment (mor1) [inline CONTRIBUTING.md]: `Databox`

File: README.md [hunk 3]
```diff
@@ -51,13 +49,11 @@ To stop the SDK run:
 
 	./databox-stop sdk
 
+## Contributing 
 
-### Develop system components and custom drivers
+The databox project welcomes contributions via pull requests see [CONTRIBUTING.md](./CONTRIBUTING.md) for more information.
```
Comment (mor1) [inline README.md]: `Databox`

File: TESTING.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      ./databox-test
+
+These test will build databox and install a driver and app pair to check 
+functionality. More work is required but it is a start. 
+
+
+# Unit tests
+
+Most core components also have unit tests these are language specific. 
+See the [core-arbitor](https://github.com/me-box/core-arbiter/tree/master/test)
```
Comment (mor1) [inline TESTING.md]: `arbitor`

**Review by mor1 — CHANGES_REQUESTED** [review]
Were these versions of files substantially copied from anywhere? If so, please check licensing and do the appropriate things.

`CONTRIBUTING.md` traditionally goes in the root dir of project (along with a few other things).

**Pull request merged**